resources/known-amendments.md

```diff
@@ -678,6 +700,26 @@ With this amendment, the issuer can now authorize the trustline.
 This amendment has no effect unless the [DisallowIncoming][] amendment is enabled.
 
 
+### fixEmptyDID
+[fixEmptyDID]: #fixemptydid
+
+| Amendment    | fixEmptyDID |
+|:-------------|:------------|
+| Amendment ID | 755C971C29971C9F20C6F080F2ED96F87884E40AD19554A5EBECDCEC8A1F77FE |
+| Status       | Open for Voting |
+| Default Vote (Latest stable release) | No |
+| Pre-amendment functionality retired? | No |
+
+This amendment adds an additional check to prevent empty DID ledger entries from being created.
+
+With this amendment, if a transaction would create an empty DID, it returns the new error code `tecEMPTY_DID` instead.
```

Comment:
not related to this PR, but: I think we should add `tecEMPTY_DID` in https://github.com/XRPLF/rippled/blob/develop/API-CHANGELOG.md

I imagine there are some developers who want their code to understand all possible error codes, so it will be useful to have mention in that API Changelog.

Reply:
Oh yeah, that should also go in the [tec codes](https://xrpl.org/docs/references/protocol/transactions/transaction-results/tec-codes) page...